app/models/manageiq/providers/openstack/cloud_manager/refresh_parser.rb

```diff
@@ -484,13 +482,13 @@ def parse_server(server, parent_hosts = nil)
       disks = new_result[:hardware][:disks]
       dev = "vda"
 
-      if (sz = flavor[:disk_size]) == 0
+      if (sz = flavor[:root_disk_size]) == 0
```

Comment:
It can go away in a separate PR or separate commit since it was in the original code.  `sz` just hurts my eyes, partly because my brain doesn't go to "size" initially; it goes to ["zero-terminated string"](https://en.wikipedia.org/wiki/Hungarian_notation#Systems_vs._Apps_Hungarian)